src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -935,4 +940,104 @@ BOOST_AUTO_TEST_CASE(CELUFunctionTest)
   CheckCELUDerivativeCorrect(desiredActivations, desiredDerivatives);
 }
 
+/**
+ * Basic test of the inverse quadratic function.
+ */
+BOOST_AUTO_TEST_CASE(InverseQuadraticFunctionTest)
+{
+  // Hand-calculated values.
+  const arma::colvec desiredActivations("0.2 0.088968 0.0470588 \
+                                         9.95913e-05 0.5 0.5 \
+                                         0.2 1");
+
+  const arma::colvec desiredDerivatives("-0.369822 -0.175152 -0.0937021 \
+                                         -0.000199183 -0.64 -0.64 -0.369822\
+                                         -0.5");
+
+  CheckActivationCorrect<InvQuadFunction>(activationData, desiredActivations);
+  CheckDerivativeCorrect<InvQuadFunction>(desiredActivations,
+                                        desiredDerivatives);
+}
+
+/**
+ * Basic test of the quadratic function.
+ */
+BOOST_AUTO_TEST_CASE(QuadraticFunctionTest)
+{
+  // Hand-calculated values using Python interpreter.
```

Comment:
The comments in the other test cases you added don't have the part of the sentence ```using Python interpreter```. Maybe we can skip this part for this test case as well?